Sources/NIO/System.swift

```diff
@@ -375,12 +376,19 @@ internal enum Posix {
     }
 
     @inline(never)
-    public static func read(descriptor: CInt, pointer: UnsafeMutableRawPointer, size: size_t) throws -> IOResult<Int> {
+    public static func read(descriptor: CInt, pointer: UnsafeMutableRawPointer, size: size_t) throws -> IOResult<ssize_t> {
```

Comment:
That's a source breaking change, I think. Why not keep it at `Int`, which is the same width and signedness with `ssize_t` on our platforms?

Reply:
This is a phantom public, public func on internal type, ie not actually public. We do have swift-api-digester running which checks api breakages.

Reply:
@vlm btw, we also sorta support 32 bit platforms (like some androids and raspberry pies)

Reply:
@weissi on 32-bits both `Int` and `ssize_t` are 32-bits.

Reply:
True, but still ssize_t is more correct here because that’s the actual return type. And this function is really not exported because the type is internal. We try to match the types as closely as we can with the actual ones in posix/libc.